ROADMAP.md

```diff
@@ -0,0 +1,31 @@
+# Style Guides Roadmap
+
+> tl;dr The goal of this document is to document the priority items for style guide development over the next 3-6 months as well as key activities designed to grow awareness and adoption of the Style Guides at STScI.
```

Comment:
3-6 months from *when*? 😉  Given the current development rate, perhaps 3-6 mo range is too aggressive?

In addition, should we also include some longer term items here?

Reply:
> 3-6 months from when? 😉 Given the current development rate, perhaps 3-6 mo range is too aggressive?

😁January 2020 --> July 2020? Although perhaps we should say this is work we want to get done in calendar year 2020?

Reply:
All I am saying is that we should be realistic. 😉